### PR TITLE
Add RSVP page for key user event

### PR DESCRIPTION
**Pull request by michellusthof**
- [x] Add page structure
- [x] Add main content
- [x] Add meta description
- [x] Add FormKeep form
- [x] Update FormKeep submission confirmation
- [x] Test FormKeep submissions

### CODE REVIEW

File: source/localizable/key-user-event.html.erb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <div class="col">
+
+      <article>
+        <h2>
+          Aanmeldformulier
+        </h2>
+
+        <%= partial "partials/form", locals: { title: "Signup: key user event",
+                                               fields: ["name", "email", "phone", "organisation_message_alt"],
+                                               multiple_persons: false,
```
Comment (RonaldDijkstra) [inline source/localizable/key-user-event.html.erb]: Probably false by default?

File: source/partials/_form.erb [hunk 2]
```diff
@@ -64,6 +64,12 @@ form_title = defined?(title) ? title : "Contact"
     </div>
   <% end %>
 
+  <% if "organisation_message_alt".in? locals[:fields] %>
```
Comment (RonaldDijkstra) [inline source/partials/_form.erb]: 🆒

File: source/localizable/key-user-event/_content.nl.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+Een groot deel van jullie werkt nu met ons nieuwe CAPP LMS. Vanuit Defacto willen we met jullie graag de nieuwste ontwikkelingen bespreken en eventuele vragen beantwoorden. Enkele gebruikers van CAPP LMS zullen daarnaast enkele interessante praktijkvoorbeelden delen.
```
Comment (RonaldDijkstra) [inline source/localizable/key-user-event/_content.nl.md]: No need for the second enkele here.

File: source/localizable/key-user-event.html.erb (hunk 1 above)
Comment (michellusthof) [reply source/localizable/key-user-event.html.erb]: So, we don't need it?

Comment (RonaldDijkstra) [reply source/localizable/key-user-event.html.erb]: Remove it see what happens

Comment (michellusthof) [reply source/localizable/key-user-event.html.erb]: Nothing, apparently.

File: source/localizable/key-user-event.html.erb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      <article>
+        <h2>
+          Aanmeldformulier
+        </h2>
+
+        <%= partial "partials/form", locals: { title: "Signup: key user event",
+                                               fields: ["name", "email", "phone", "organisation_message_alt"],
+                                               button_text: "Aanmelden",
+                                               action_url: "https://formkeep.com/f/99d8b5db9d3e"} %>
+      </article>
```
Comment (RonaldDijkstra) [inline source/localizable/key-user-event.html.erb]: Enter after article.

**Pull request merged**